### DISCUSSION TITLE
Concurrent Recurrent Jobs limitations

### DISCUSSION THREAD

**Discussion by sonjoydabnath** [jobrunr/jobrunr#799] [Q&A]
Hi everyone!
As per the documentation, it says the free and open-source version of JobRunr only allows 100 recurring jobs,

![image](https://github.com/jobrunr/jobrunr/assets/9343674/e0c2665b-17e0-4dea-bbac-f276d7b2faac)

But I am able to run more than 100 recurring jobs. I am using a background server and 128 workers configuration.

![image](https://github.com/jobrunr/jobrunr/assets/9343674/134f65b1-de16-4eb3-b2fd-13a29fd79bcd)

Please help me understand what the limitation of 100 recurring jobs in the documentation actually means.

**Comment by rdehuyss** [comment]
Please search the docs. This has been answered again and again. Also see https://github.com/jobrunr/jobrunr/discussions/769
